larray/core/session.py

```diff
@@ -1036,7 +1037,12 @@ def opmethod(self, other):
                             except Exception:
                                 res_item = nan
                     res.append((name, res_item))
-            return Session(res)
+            try:
+                # XXX: print a warning?
```

Comment:
I think we had discussed this in the issue or this PR but I can't remember what the outcome was.

Reply:
The discussion starts [here](https://github.com/larray-project/larray/issues/832#issuecomment-610850615) in the corresponding issue thread #832.

The outcome was to [let it as it is implemented now](https://github.com/larray-project/larray/issues/832#issuecomment-618399372).